cf-usb/tasks/service-commit-sources.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This script publishes the cf-usb-sidecar bundle
+# Most arguments come from the task definition
+
+set -o errexit
+
+if [[ "${COMMIT_SOURCES}" != "true" ]]; then
+  echo "+------------------------------------------------+"
+  echo "| On openSUSE builds no source will be published |"
```

Comment:
This message can be confusing if the flag is set to `false` for non-openSUSE builds in the future.